### DISCUSSION TITLE
Automatic fallback to generic editor

### DISCUSSION THREAD

**Discussion by AtmanActive** [apohl79/audiogridder#407] [Ideas]
There is a use case where Audiogridder fails to do the right thing: when working mostly with UI-based plugins, but then throwing in an UI-less plug-in, like [Airwindows](https://www.airwindows.com), for example. In such a situation, Audiogridder just looks like it is failing, unable to show any controls. Only when an end-user switches to Generic Editor manually does the Audiogridder understand what should be done.

Hence the question: can Audiogridder-server detect that a plug-in has no dedicated UI and inform the Audiogridder-plugin about it and have it automatically switch to Generic Editor? 
Or maybe just have the server display a dummy UI that says something like "this plug-in has no UI, please switch to Generic Editor".

**Comment by apohl79** [comment]
Thats a great idea, I will create a feature request for it. I like switching to the generic editor automatically.
